tsmo/src/status_intent_plotter.py

```diff
@@ -60,7 +60,7 @@ def status_intent_plotter():
 
         plt.xticks(rotation=75)
         axs=plt.gca()
-        xfmt = md.DateFormatter('%H:%M:%S.%f') 
+        xfmt = md.DateFormatter('%H:%M:%S') 
```

Comment:
Did you remove the decimal accuracy? This is fine as long as you don't have sequential ticks that look like the represent the same time.

Reply:
Removed them per @Saeid-Solei request. Yes there aren't any of those kind of ticks.